### DISCUSSION TITLE
Where are they?

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#458] [Q&A] [answered]
I can't find two things anymore...
1. place waypoints on the road
2.How do I save my planned routes in "My Places"

**Comment by devemux86** [comment, marked as answer]
Both are in the same sub-menu as before with the new name "Options...".

**Reply by Leeb64** [reply]
2x options and 2x the same symbol.
a little confusing!
Maybe you could rename the options under "Routing" to Settings?

**Reply by devemux86** [reply]
Better leave the "Options" in "Routing" as they are indeed the options of the selected router.

Probably we can use a different icon + name in the route menu.
And similarly the same in the menus of favorites and waypoints.

**Comment by devemux86** [comment]
Let's continue the icon / text discussion here:
- https://github.com/devemux86/cruiser/discussions/444
